[2024-12-09T01:20:37.189+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-09T01:20:37.202+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_dag.task_ingest_statuses_data manual__2024-12-09T01:19:14.466321+00:00 [queued]>
[2024-12-09T01:20:37.208+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_dag.task_ingest_statuses_data manual__2024-12-09T01:19:14.466321+00:00 [queued]>
[2024-12-09T01:20:37.209+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2024-12-09T01:20:37.220+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): task_ingest_statuses_data> on 2024-12-09 01:19:14.466321+00:00
[2024-12-09T01:20:37.226+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=936) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-09T01:20:37.227+0000] {standard_task_runner.py:72} INFO - Started process 938 to run task
[2024-12-09T01:20:37.227+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_dag', 'task_ingest_statuses_data', 'manual__2024-12-09T01:19:14.466321+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp1sdk_52_']
[2024-12-09T01:20:37.230+0000] {standard_task_runner.py:105} INFO - Job 51: Subtask task_ingest_statuses_data
[2024-12-09T01:20:37.266+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_dag.task_ingest_statuses_data manual__2024-12-09T01:19:14.466321+00:00 [running]> on host 58979770e908
[2024-12-09T01:20:37.339+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_dag' AIRFLOW_CTX_TASK_ID='task_ingest_statuses_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-09T01:19:14.466321+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-09T01:19:14.466321+00:00'
[2024-12-09T01:20:37.340+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-09T01:20:37.352+0000] {execute.py:83} INFO - Input Notebook:  /opt/***/dataset/cleanup_statuses.ipynb
[2024-12-09T01:20:37.353+0000] {execute.py:84} INFO - Output Notebook: /opt/***/dataset/output_cleanup_statuses.ipynb
[2024-12-09T01:20:37.388+0000] {execute.py:97} WARNING - Passed unknown parameter: file_path
[2024-12-09T01:20:37.389+0000] {execute.py:97} WARNING - Passed unknown parameter: sheet_name
[2024-12-09T01:20:37.394+0000] {parameterize.py:114} WARNING - Input notebook does not contain a cell with tag 'parameters'
[2024-12-09T01:20:37.405+0000] {logging_mixin.py:190} WARNING - Executing:   0%|          | 0/4 [00:00<?, ?cell/s]
[2024-12-09T01:20:39.019+0000] {clientwrap.py:44} INFO - Executing notebook with kernel: python3
[2024-12-09T01:20:39.042+0000] {logging_mixin.py:190} WARNING - Executing:  25%|##5       | 1/4 [00:01<00:04,  1.64s/cell]
[2024-12-09T01:20:39.349+0000] {logging_mixin.py:190} WARNING - Executing:  50%|#####     | 2/4 [00:01<00:01,  1.17cell/s]
[2024-12-09T01:20:46.164+0000] {logging_mixin.py:190} WARNING - Executing:  75%|#######5  | 3/4 [00:08<00:03,  3.58s/cell]
[2024-12-09T01:20:46.541+0000] {logging_mixin.py:190} WARNING - Executing: 100%|##########| 4/4 [00:09<00:00,  2.31s/cell]
[2024-12-09T01:20:46.864+0000] {logging_mixin.py:190} WARNING - Executing: 100%|##########| 4/4 [00:09<00:00,  2.36s/cell]
[2024-12-09T01:20:46.991+0000] {base.py:84} INFO - Retrieving connection 'external_postgres'
[2024-12-09T01:20:49.153+0000] {logging_mixin.py:190} INFO - Data from sheet 'Statuses' loaded into table 'l0_statuses' after cleanup.
[2024-12-09T01:20:49.156+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-09T01:20:49.165+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-09T01:20:49.166+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_ingestion_dag, task_id=task_ingest_statuses_data, run_id=manual__2024-12-09T01:19:14.466321+00:00, execution_date=20241209T011914, start_date=20241209T012037, end_date=20241209T012049
[2024-12-09T01:20:49.209+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-09T01:20:49.221+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-09T01:20:49.224+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
